TO-DO-LIST WEB app/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Task List - SJsquare</title>
+    <link rel="icon" href="SJsquare Logo.png" type="image/png">  
+
+	<link rel="stylesheet" href="main.css" />
+</head>
+<body>
+	
+	<header>
+       
+		<h1><span>TO DO LIST</span> WEB APP</h1>
+        
+		<form id="new-task-form">
+			<input 
+				type="text" 
+				name="new-task-input" 
+				id="new-task-input" 
+				placeholder="What do you have planned?" />
+			<input 
+				type="submit"
+				id="new-task-submit" 
+				value="Add task" />
+		</form>
+	</header>
+	<main>
+		<section class="task-list">
+			<h2>WORKS</h2>
+
+			<div id="tasks">
+
+				<!-- <div class="task">
```

Comment:
Why have you commented out this portion of code

Reply:
the reason when we add a task below that it shows the work in which edit and delete button becomes visible and we will not comment out then edit and delete button will not be invisible if we delete a task. So, it is necessary to comment out that to make it run properly